app/src/main/java/com/td/mylevelup/mortgage/mortgageCalculatorPage/MortgagePageAdapter.kt

```diff
@@ -2,9 +2,9 @@ package com.td.mylevelup.mortgage.mortgageCalculatorPage
 
 import com.ngam.rvabstractions.general.AbstractAdapter
 
-class MortgagePageAdapter(private val presenter: MortgagePagePresenter): AbstractAdapter() {
+class MortgagePageAdapter(private val presenter: MortgagePagePresenter) : AbstractAdapter() {
```

Comment:
extra space. Should be `MortgagePageAdapter(...): AbstractAdapter()` without the space